src/app/pdf-viewer/pdf-viewer.component.scss

```diff
@@ -14,487 +15,384 @@
-      margin: 1px auto -8px auto;
-      position: relative;
-      overflow: visible;
-      // https://stackoverflow.com/questions/21025212/border-image-not-showing-in-safari
-      // "transparent" makes the border not show up in Safari. But adding a light transparent color did
-      // rgba(0,0,0,0.01). But there needs to be an alpha value larger than 0.
-      border: 9px solid rgba(0,0,0,0.01);
-      background-clip: content-box;
-      -o-border-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABMAAAATCAQAAADYWf5HAAAA6UlEQVR4Xl2Pi2rEMAwE16fm1f7/r14v7w4rI0IzLAF7hLxNevBSEMEF5+OilNCsRd8ZMyn+a4NmsOT8WJw1lFbSYgGFzF2bLFoLjTClWjKKGRWpDYAGXUnZ4uhbBUzF3Oe/GG/ue2fn4GgsyXhNgysV2JnrhKEMg4fEZcALmiKbNhBBRFpSyDOj1G4QOVly6O1FV54ZZq8OVygrciDt6JazRgi1ljTPH0gbrPmHPXAbCiDd4GawIjip1TPh9tt2sz24qaCjr/jAb/GBFTbq9KZ7Ke/Cqt8nayUikZKsWZK7Fe6bg5dOUt8fZHWG2BHc+6EAAAAASUVORK5CYII=') 9 9 repeat;
-      border-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABMAAAATCAQAAADYWf5HAAAA6UlEQVR4Xl2Pi2rEMAwE16fm1f7/r14v7w4rI0IzLAF7hLxNevBSEMEF5+OilNCsRd8ZMyn+a4NmsOT8WJw1lFbSYgGFzF2bLFoLjTClWjKKGRWpDYAGXUnZ4uhbBUzF3Oe/GG/ue2fn4GgsyXhNgysV2JnrhKEMg4fEZcALmiKbNhBBRFpSyDOj1G4QOVly6O1FV54ZZq8OVygrciDt6JazRgi1ljTPH0gbrPmHPXAbCiDd4GawIjip1TPh9tt2sz24qaCjr/jAb/GBFTbq9KZ7Ke/Cqt8nayUikZKsWZK7Fe6bg5dOUt8fZHWG2BHc+6EAAAAASUVORK5CYII=') 9 9 repeat;
-      background-color: white;
-    }
-
-    &.removePageBorders .page {
-      margin: 0px auto 10px auto;
-      border: none;
-    }
-    
-    padding-bottom: 10px;
-    &.removePageBorders {
-      padding-bottom: 0;
-    }
-
-    &.singlePageView {
-      display: inline-block;
-
-      .page {
-        margin: 0;
-        border: none;
-      }
-    }
-
-    &.scrollHorizontal,
-    &.scrollWrapped {
-      margin-left: 3.5px;
-      margin-right: 3.5px;
-      text-align: center;
-    }
-  }
-
-  .spread {
+  .annotationLayer .popup span {
+    display: inline-block;
+    margin-left: 5px;
+  }
+
+  .annotationLayer .popup p {
+    border-top: 1px solid #333;
+    margin-top: 2px;
+    padding-top: 2px;
+  }
+
+  .annotationLayer .highlightAnnotation,
+  .annotationLayer .underlineAnnotation,
+  .annotationLayer .squigglyAnnotation,
+  .annotationLayer .strikeoutAnnotation,
+  .annotationLayer .freeTextAnnotation,
+  .annotationLayer .lineAnnotation svg line,
+  .annotationLayer .squareAnnotation svg rect,
+  .annotationLayer .circleAnnotation svg ellipse,
+  .annotationLayer .polylineAnnotation svg polyline,
+  .annotationLayer .polygonAnnotation svg polygon,
+  .annotationLayer .caretAnnotation,
+  .annotationLayer .inkAnnotation svg polyline,
+  .annotationLayer .stampAnnotation,
+  .annotationLayer .fileAttachmentAnnotation {
+    cursor: pointer;
+  }
+
+  .pdfViewer .canvasWrapper {
+    overflow: hidden;
+  }
+
+  .pdfViewer .page {
+    direction: ltr;
+    width: 816px;
+    height: 1056px;
+    margin: 1px auto -8px auto;
+    position: relative;
+    overflow: visible;
+    border: 9px solid transparent;
```

Comment:
you removed a bug fix for the borders for safari browsers. the original line was:
```
// https://stackoverflow.com/questions/21025212/border-image-not-showing-in-safari
// "transparent" makes the border not show up in Safari. But adding a light transparent color did
// rgba(0,0,0,0.01). But there needs to be an alpha value larger than 0.
border: 9px solid rgba(0,0,0,0.01);
```